src/utils/dataHandler.js

```diff
@@ -37,8 +37,26 @@ function getClassifications() {
     return classifications
 }
 
+// The function to go to the upper page
+function goUpperPage(classification) {
+    window.location.pathname = 'roles/' + classification + 'Roles'
+}
+
+// The function for navigation through roles pages
+function shiftRolesPage(index, x){
+    
+    const newIndex = index + x
```

Comment:
missing semi-colon at the end. maybe it's time to install a eslint, prettier and other linting tools to make your project clean and standard

Reply:
of course ❤️ 💯